src/tests/ftest/server/metadata.py

```diff
@@ -16,21 +15,22 @@
 from thread_manager import ThreadManager
 
 
-def run_ior_loop(manager, uuids):
-    """IOR run for each UUID provided.
+def run_ior_loop(test, manager, num_run):
+    """Run IOR multiple times.
 
     Args:
         manager (str): mpi job manager command
-        uuids (list): list of container UUIDs
+        num_run (int): number of times to run IOR
 
     Returns:
         list: a list of CmdResults from each ior command run
 
     """
     results = []
     errors = []
-    for index, cont_uuid in enumerate(uuids):
-        manager.job.dfs_cont.update(cont_uuid, "ior.cont_uuid")
+    for index in range(num_run):
+        cont = test.label_generator.get_label('cont')
```

Comment:
`TestContainer`?

Reply:
Actually, I was wrong. This is not supported by IOR:
```
ior -a DFS --dfs.pool <pool>
```
A container is required, whether we set DAOS_UNS_PREFIX or not:
```

ior -a DFS --dfs.pool <pool> --dfs.cont <cont>
```


So the only dilemma here is whether we want to create the container upfront in the test, or generate a label and let IOR create it. I'm on the fence